plugins/aws/ec2/openMemcached.js

```diff
@@ -9,13 +9,19 @@ module.exports = {
     more_info: 'While some ports such as HTTP and HTTPS are required to be open to the public to function properly, more sensitive services such as Memcached should be restricted to known IP addresses.',
     link: 'http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/authorizing-access-to-an-instance.html',
     recommended_action: 'Restrict TCP and UDP port 11211 to known IP addresses',
-    apis: ['EC2:describeSecurityGroups', 'EC2:describeNetworkInterfaces', 'Lambda:listFunctions'],
+    apis: ['EC2:describeSecurityGroups', 'EC2:describeNetworkInterfaces', 'Lambda:listFunctions', 'EC2:describeSubnets', 'ElastiCache:describeCacheClusters','ElastiCache:describeCacheSubnetGroups' ],
     settings: {
         ec2_skip_unused_groups: {
             name: 'EC2 Skip Unused Groups',
             description: 'When set to true, skip checking ports for unused security groups and produce a WARN result',
             regex: '^(true|false)$',
             default: 'false',
+        },
+        private_clusters :{
+            name: 'Memcached Private Cluster Associated Groups',
+            description: 'When set to true, checks if the security group has all the private cluster associated',
+            regex: '^(true|false)$',
+            default: 'false',
         }
```

Comment:
make it more descriptive, something like
```suggestion
        ignore_groups_with_private_clusters :{
            name: 'Security Groups with Private Memcached Clusters',
            description: 'When set to true, passes if all the clusters associated with a security group are in a private subnet',
            regex: '^(true|false)$',
            default: 'false',
        }
```

Reply:
Fix the failing tests